documentation/docs/release-notes/2.44.0_1.md

```diff
@@ -0,0 +1,116 @@
+
+| Affected deployments                     | Version | Notes |
+|-------------------------------------------|---------|-------|
+| PMM Open Virtual Appliance (OVA) installations | ≥ 2.38  |       |
+
+### Mitigation options  
+
+
+=== "PREFERRED: Upgrade to PMM 2.44.0-1"
+    This release enhances security in OVA deployments by automatically removing unnecessary system accounts during the initial setup.
+
+    To secure your system, follow these steps to upgrade:
+
+    1. Download and deploy the new OVA file from [Percona website](https://www.percona.com/downloads).
+
+    2. Stop services on your current installation: 
+        ```sh 
+        supervisorctl stop all
+        ```
+    3. Back up and transfer your data:
+        ```sh
+        cd /srv
+        tar -cvf srv.tar .
+        ``` 
+    4. Transfer to new server.
+    5. Deploy data on the new installation:
+        ```sh
+        # Stop all services
+        supervisorctl stop all
+
+        # Clear existing data
+        cd /srv
+        rm -rf *
+
+        # Extract backup
+        tar -xvf srv.tar
+
+        # Start all services
+        supervisorctl start all
+        ```
+    6. Update DNS records or swap IP addresses to ensure uninterrupted Client connectivity.
+
+    #### Verification steps
+
+    After upgrading, verify that your system is functioning correctly:
+
+    1. Check service status to confirm both PMM Client and PMM Server are running:
+        ```sh
+        supervisorctl status
+        ```
+    2. Ensure Client connectivity to validate data flow.
+
+    3. Test system functionality by performing basic monitoring tasks.
+
+=== "TEMPORARY: If you cannot upgrade immediately"
+    If an upgrade is not immediately possible, follow these steps to mitigate the vulnerability:
+
+    1. Block port 22 access.
+    2. Execute ONE of these commands to secure the vulnerable account:
+
+        - Disable login:
+            ```sh
+            sudo usermod -s /sbin/nologin vagrant
+            ```
+
+        - Lock account:
+            ```sh
+            sudo passwd -l vagrant
+            ```
+
+        - Remove user completely:
+            ```sh
+            sudo kill -9 $(pgrep -f vagrant)
+            sudo userdel -r vagrant
+            ```
+    3. Change all service passwords.
+    4. Monitor system logs for unauthorized access.
+
+### Support & additional resources
+If you require further clarification or assistance, we are available to assist you 24/7:
+
+ - [Technical Support Portal](https://my.percona.com)
+ - [Security advisory updates][blog address from Eduardo]
+ - Emergency contact: [any specific contact details?]
+
+### Upgrading from PMM 3.0.0?
+If you are upgrading from PMM 3.0.0, see the separate [PMM 3.0.0-1 release notes](https://docs.percona.com/percona-monitoring-and-management/3/release-notes/3.0.0_1.html) for specific upgrade instructions.
```

Comment:
it's not possible to upgrade from 3.0.0 to 2.44.0-1, only to 3.0.0-1. I think it should be rephrased.